main.go

```diff
@@ -82,6 +82,672 @@ var (
 	masterMetrics = []snapshotMetric{
 		snapshotMetric{
 			desc: prometheus.NewDesc(
+				"mesos_master_cpus_percent",
+				"Percent of CPUs used as a fraction of 1.",
```

Comment:
Is it a fraction or a percentage?

We prefer ratios, as it avoids confusion. The raw values are even better.